gems/pending/appliance_console.rb

```diff
@@ -38,7 +38,7 @@
 
 [:INT, :TERM, :ABRT, :TSTP].each { |s| trap(s) { raise MiqSignalError } }
 
-RAILS_ROOT    = Pathname.new("#{ROOT}/vmdb")
+RAILS_ROOT    = Pathname.new("#{ROOT}")
```

Comment:
thanks - didn't we change away from `/var/www/miq/vmdb`
Also, isn't the Gemfile now at root?